test/hotspot/jtreg/vmTestbase/nsk/jdi/ThreadReference/stop/stop002.java

```diff
@@ -54,25 +59,31 @@ public class stop002 {
     static final String DEBUGGEE_THRNAME = "stop002tThr";
 
     // debuggee local var used to find needed non-throwable object
-    static final String DEBUGGEE_LOCALVAR = "stop002tNonThrowable";
-    // debuggee field used to indicate that testing is over
-    static final String DEBUGGEE_FIELD = "stopLooping";
+    static final String DEBUGGEE_NON_THROWABLE_VAR= "stop002tNonThrowable";
+    // debuggee local var used to find needed non-throwable object
```

Comment:
This comment is a little bit confusing.
Should it be one line up or it is intentionally placed before `DEBUGGEE_THROWABLE_VAR`?

Reply:
It should be "throwable", not "non-throwable"